docs/content/kafka-authorization.md

```diff
@@ -115,43 +142,77 @@ example below shows the input structure.
 
 ```json
 {
-  "operation": {
-    "name": "Write"
-  },
-  "resource": {
-    "resourceType": {
-      "name": "Topic"
+  "action": {
+    "logIfAllowed": true,
+    "logIfDenied": true,
+    "operation": "WRITE",
+    "resourcePattern": {
+      "name": "credit-scores",
+      "patternType": "LITERAL",
+      "resourceType": "TOPIC",
+      "unknown": false
     },
-    "name": "credit-scores"
+    "resourceReferenceCount": 1
   },
-  "session": {
+  "requestContext": {
+    "clientAddress": "/172.22.0.1",
+    "clientInformation": {
+      "softwareName": "apache-kafka-java",
+      "softwareVersion": "2.8.1"
+    },
+    "connectionId": "172.22.0.5:9093-172.22.0.1:62744-2",
+    "header": {
+      "headerVersion": 2,
+      "name": {
+        "clientId": "consumer-console-producer-63933-1",
+        "correlationId": 10,
+        "requestApiKey": 1,
+        "requestApiVersion": 12
+      }
+    },
+    "listenerName": "SSL",
     "principal": {
+      "name": "CN=anon_producer,OU=Developers",
       "principalType": "User"
     },
-    "clientAddress": "172.21.0.5",
-    "sanitizedUser": "CN%3Danon_producer.tutorial.openpolicyagent.org%2COU%3DTUTORIAL%2CO%3DOPA%2CL%3DSF%2CST%3DCA%2CC%3DUS"
+    "securityProtocol": "SSL"
   }
 }
 ```
 
 The client identity is extracted from the SSL certificates that clients
-present when they connect to the broker. The client identity information is
-encoded in the `input.session.sanitizedUser` field. This field can be decoded
-inside the policy.
+present when they connect to the broker. The user identity information is
+encoded in the `input.requestContext.principal.name` field.
+This field can be used inside the policy.
 
 Generating SSL certificates and JKS files required for SSL client
-authentication is outside the scope of this tutorial. To simplify the steps
-below, the Docker Compose file uses an extended version of the
-[confluentinc/cp-kafka](https://hub.docker.com/r/confluentinc/cp-kafka/)
-image from Docker Hub. The extended image includes **pre-generated SSL
-certificates** that the broker and clients use to identify themselves.
+authentication is outside the scope of this tutorial, but the plugin
```

Comment:
I'd provide a cURL command to download the script and then tell people to run it. I had to fish around and then realize that the script happened to generate the files under the location expected by the compose file.

Reply:
Fixing 👍